src/newfight.cpp

```diff
@@ -1179,6 +1186,33 @@ void hit_with_multiweapon_toggle(struct char_data *attacker, struct char_data *v
     }
   }
 
+  // Handle spirits and elementals being little divas with their special combat rules.
```

Comment:
This block should have replaced the previous one-- search for 'divas' and plop it in on top of that old block. Down here, it doesn't really affect anything.